http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcProviderMetadataTest.java

```diff
@@ -0,0 +1,61 @@
+package org.wildfly.security.http.oidc;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test OIDC json config class properly returns boolean values.
+ */
+public class OidcProviderMetadataTest {
+    private OidcProviderMetadata oidcSecRealm;
```

Comment:
Just minor, but since this represents the provider metadata, I think oidcProviderMetadata would be better than oidcSecRealm.

Reply:
done